cmd/dfdaemon/app/init.go

```diff
@@ -70,7 +70,7 @@ func initDfdaemon(cfg config.Properties) error {
 	return nil
 }
 
-// initLogger initialize the global logrus logger
+// initLogger initializes the global logrus logger
```

Comment:
Why do you don't add . here?

Reply:
![image](https://user-images.githubusercontent.com/19339970/65103917-4d1c4980-da02-11e9-97c5-efa361cf6c32.png)

Reply:
// initLogger initializes the global logrus logger
func initLogger(cfg config.Properties) error {
	current, err := user.Current()
	if err != nil {
		return errors.Wrap(err, "get current user")
	}

	logFilePath := filepath.Join(current.HomeDir, ".small-dragonfly/logs/dfdaemon.log")

	opts := []dflog.Option{
		dflog.WithLogFile(logFilePath),
		dflog.WithSign(fmt.Sprintf("%d", os.Getpid())),
		dflog.WithDebug(cfg.Verbose),
		dflog.WithConsole(),
	}

	logrus.Debugf("use log file %s", logFilePath)

	return errors.Wrap(dflog.Init(logrus.StandardLogger(), opts...), "init log")
}

Reply:
Function